warehouse/warehouse_test.go

```diff
@@ -0,0 +1,59 @@
+package warehouse_test
```

Comment:
Consider renaming this file to scheduling_test.go ?

Reply:
renamed